modules/hedera-app-api/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2020-2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id("com.hedera.hashgraph.conventions")
+}
+
+description = "Hedera Application API"
+
+dependencies {
+    compileOnlyApi("org.jetbrains:annotations:20.1.0")
```

Comment:
Only question here is which library to use. Could be JetBrains, or could be the JSR jar we're already using. The question will be, does spotbugs work with the jetbrains annotations? What about other static analysis tools like ErrorProne?

Reply:
discussed in https://github.com/hashgraph/hedera-services/issues/4234